src/Polar/kernels.jl

```diff
@@ -1,4 +1,5 @@
 import KernelAbstractions: @index
+using ForwardDiff
```

Comment:
I don't think we need to import `ForwardDiff` here, as it is already loaded in `src/ExaPF.jl`

Reply:
ok!